<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessStudent</class>
 <widget class="QDialog" name="ProcessStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProcessStudent</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Student Number: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="studentNumberLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="displayButton">
       <property name="text">
        <string>Display</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="averageButton">
       <property name="text">
        <string>Average</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="graduateButton">
       <property name="text">
        <string>Graduate?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addModuleButton">
       <property name="text">
        <string>Add module</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
